### ISSUE TITLE
[BUG] Testing a Persistent Actor on .NET 4.5

### ISSUE THREAD

**Issue by nsanitate** [akkadotnet/akka.net#3325]
I want to learn how to use Persistent Actor, but I don't understand why my implemented scenario (a ping-pong like messages exchange) raises a timeout exception in my NUnit test waiting for a response.

If the actor extends a non-persistent ReceiveActor and use `Receive` instead of `Command`, the test turns green.

Here the actor class.
```c#
public class FakeActor : ReceivePersistentActor
{
    public override string PersistenceId { get; } = "HashCoded";

    public FakeActor()
    {
        Command<FakeRequest>(request => Sender.Tell(new FakeResponse(requestId: request.RequestId)));
    }
}
```

Here the actor test class
```c#
public class FakeActorTest : TestKit
{
    [Test]
    public void FakeActor_FakeRequest()
    {
        // Arrange
        var senderProbe = CreateTestProbe();
        var sut = Sys.ActorOf(Props.Create<FakeActor>());

        // Act
        sut.Tell(new FakeRequest(1), senderProbe.Ref);

        // Assert
        senderProbe.ExpectMsg<FakeResponse>(r => r.RequestId == 1);
    }
}
```

And here the protocol classes.
```c#
public class FakeRequest
{
    public FakeRequest(ulong requestId) { RequestId = requestId; }
    public override string ToString() => $"FakeRequest: {RequestId}";
    public ulong RequestId { get; private set; }
}
```

```c#
public class FakeResponse
{
    public FakeResponse(ulong requestId) { RequestId = requestId; }
    public override string ToString() => $"FakeResponse: {RequestId}";
    public ulong RequestId { get; private set; }
}
```

Specs:
* Which Akka.Net version you are using: 1.3.2
* On which platform you are using Akka.Net: Windows - .NET 4.5

**Comment by nsanitate** [comment]
UPDATE: It works on .NET Core

**Comment by gztamas** [comment]
I tried to reproduce the issue, but failed.
The test runs green in my solution. Win10, .NET 4.5, Akka 1.3.2. I use NUnit3 for the tests. I used VS2015 for creating the solution. The code samples are from this bug report, without change (just added the necessary usings and namespaces). The sample solution is uploaded to https://github.com/gztamas/akkaNetSample3325

Please check what's the difference between your solution and mine.

**Comment by gztamas** [comment]
Oh, I got it. I recalled that I had a similar issues previously. For some reason I need to run Visual Studio in administrator mode for the actor tests to work when using persistent actors. I tried running the test in non-administrator VS2015 and it indeed timed out. I restarted VS in administrator mode and the code passed.

I remember now, back then I spent hours trying to find out why almost all of my tests broke suddenly. In the end I decided to restart VS, and after that the tests passed. I was puzzled, it took me a while to realize it was because VS starts in administrator mode by default if I start it normally, but when the tests broke it was not running in that mode for some reason.
